### PR TITLE
Happy case

### PR DESCRIPTION
**Pull request by valzugg**
Ends the questionnaire with a 'yes', also POSTing happens.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -62,16 +66,42 @@ class App extends React.Component {
...
+        return currentQuestionIndex !== questions.length - 1 && answers !== undefined &&
+            // finds the question which depends on the given answer
+            questions.find(question => question.depends_on_question_id === questions[currentQuestionIndex].id).
+            depends_on_choice_id === answers[questions[currentQuestionIndex].id].id
+    }
+
+
     submitObservation = () => {
-        // TODO POST observation
+        // NOTE: place and category still need to be fetched from API
```
Comment (VonLatvala) [inline src/App.jsx]: please change this `NOTE` to `TODO` or `FIXME`

File: src/service.js [hunk 2]
```diff
@@ -14,7 +14,11 @@ const getChoices = async (id) => {
 };
 
 const postObservation = async (data) => { // eslint-disable-line
-    // TODO
+    axios.post('https://app-staging.incy.io/api/bitsaber-staging/observations/links/staging-place-tarvikkeet', data)
+        .then(res => {
+            console.log(res)
```
Comment (VonLatvala) [inline src/service.js]: does this compile? Linter should complain about logging

File: src/App.jsx [hunk 3]
```diff
@@ -62,16 +66,42 @@ class App extends React.Component {
         const { currentQuestionID, questions } = this.state;
         const currentQuestionIndex = questions.findIndex(question => question.id === currentQuestionID);
 
-        if (currentQuestionIndex !== questions.length - 1) { // more questions
+        if (this.moreQuestions(currentQuestionIndex)) { // more questions
```
Comment (VonLatvala) [inline src/App.jsx]: Pretty cool improvement tbh fam

File: src/service.js (hunk 2 above)
Comment (valzugg) [reply src/service.js]: So I should remove console.logs? Compiles fine

File: src/App.jsx (hunk 3 above)
Comment (valzugg) [reply src/App.jsx]: tack!

File: src/service.js (hunk 2 above)
Comment (VonLatvala) [reply src/service.js]:
```
ERROR in ./src/App.jsx
Module Error (from ./node_modules/eslint-loader/index.js):

/home/axel/Documents/dev/incy-io-kiosk-frontend/src/App.jsx
  85:1  error  Expected indentation of 16 spaces but found 12  indent

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

 @ ./src/index.jsx 5:0-24 6:118-121
 @ multi (webpack)-dev-server/client?http://0.0.0.0:3000 (webpack)/hot/dev-server.js ./src/index.jsx

ERROR in ./src/service.js
Module Error (from ./node_modules/eslint-loader/index.js):

/home/axel/Documents/dev/incy-io-kiosk-frontend/src/service.js
  19:13  error  Unexpected console statement  no-console
  20:13  error  Unexpected console statement  no-console
  21:27  error  Unexpected console statement  no-console

✖ 3 problems (3 errors, 0 warnings)
```
Does not compile for me

Comment (VonLatvala) [reply src/service.js]: But you can bypass these by adding `--eslint-disable-line` after each `console.log()` for now, but never into master

Comment (VonLatvala) [reply src/service.js]: I should probably remove this rule for development environment...

**Pull request merged**